specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1343,6 +1768,262 @@
       "type": "object",
       "properties": {},
       "description": "Request object for NSP reconcile."
+    },
+    "NspLink": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NspLinkProperties",
+          "description": "Properties of the network security perimeter link"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the link resource that is unique within a perimeter. This name can be used to access the resource."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Identifier of the network security perimeter link in ARM id format."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
```

Comment:
>ef": "./network.json#/definitions/ProxyResource" [](http://example.com/codeflow?start=13&length=48)

Doesnt this already have the properties for name \ id \ type ?

Reply:
Thanks removed it.